public/app/core/components/AppChrome/AppChrome.tsx

```diff
@@ -32,6 +32,9 @@ export function AppChrome({ children }: Props) {
   const dockedMenuBreakpoint = theme.breakpoints.values.xl;
   const dockedMenuLocalStorageState = store.getBool(DOCKED_LOCAL_STORAGE_KEY, true);
   const menuDockedAndOpen = !state.chromeless && state.megaMenuDocked && state.megaMenuOpen;
+  // TODO how to get this state from the scene?
+  const scopeDashboardsState = true;
```

Comment:
long term this should be part of app chrome state, and be something like "leftSideToolbarPane", and made a more generic app chrome feature so we do not need a hard dependency on DashoardScopes from AppChrome.

Reply:
Two things to have in mind:
- scopes might be moved to enterprise but this is yet to be decided later
- I know @ashharrison90 is working on some changes that involve the menu and the breadcrumbs bar. We're waiting for that to happen before @Ijin08 can properly place the dashboards pane in the UI.